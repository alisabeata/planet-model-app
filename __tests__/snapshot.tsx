/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react'
import Layout from '@/app/layout'
import Page from '@/app/page'

it('render Layout', () => {
  const { container } = render(<Layout>Test</Layout>)
  expect(container).toMatchSnapshot()
})
