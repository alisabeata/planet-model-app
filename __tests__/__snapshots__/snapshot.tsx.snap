// Jest Snapshot v1, https://goo.gl/fbAQLP
// 'npm run test -- -u' to update snapshot

exports[`render Layout 1`] = `
<div>
  <html
    lang="en"
  >
    <body>
      Test
    </body>
  </html>
</div>
`;
